app/controllers/reports/regions_controller.rb

```diff
@@ -124,12 +124,8 @@ def set_facility_keys
     @facility_keys = [district, *facilities]
   end
 
-  def default_period
-    Period.month(Date.current.last_month.beginning_of_month).attributes
-  end
-
   def set_period
-    period_params = report_params[:period].presence || default_period
+    period_params = report_params[:period].presence || Reports::RegionService.default_period.attributes
```

Comment:
Going to use the one `default_period` everywhere now.

This means that region reports will now show the current months data when there is no period selected, which should be better for performance reasons because that is what we are caching in Region cache warmer.